PostgreSQL/connector.js

```diff
@@ -0,0 +1,29 @@
+const { Client } = require('pg');
+
+let logger = global.logger;
+
+module.exports = async (connectorData) => {
+	logger.trace("PostgreSQL Connector: Invoked!");
+	logger.trace(`PostgreSQL Connector: Connector Data: ${JSON.stringify(connectorData)}`);
+	logger.trace(`PostgreSQL Connector: Connecting to PostgreSQL: ${connectorData.connectionString}`);
+	try {
+		const client = new Client({
+			user: connectorData.user,
+			password: connectorData.password,
+			host: connectorData.host,
+			port: connectorData.port,
+			database: connectorData.database,
+		});
+		await client.connect()
+		logger.info("PostgreSQL Connector: Connected to PostgreSQL");
+		logger.info(`PostgreSQL Connector: DB set to ${connectorData.database}`)
+		return { pgCon: client };
```

Comment:
Return a client `return { client: client, db: connectorData.database };` and not `return { pgCon: client };`.

Let's be consistent how we return connection objects.

Reply:
Fixed.